helpers/round_changes.js

```diff
@@ -64,18 +64,20 @@ function RoundChanges(scope) {
 RoundChanges.prototype.at = function(index) {
 	var fees = new Bignum(this.roundFees.toPrecision(15))
 		.dividedBy(slots.delegates)
-		.floor();
+		.integerValue(Bignum.ROUND_FLOOR);
 	var feesRemaining = new Bignum(this.roundFees.toPrecision(15)).minus(
-		fees.times(slots.delegates)
+		fees.multipliedBy(slots.delegates)
 	);
 	var rewards =
-		new Bignum(this.roundRewards[index].toPrecision(15)).floor() || 0;
+		new Bignum(this.roundRewards[index].toPrecision(15)).integerValue(
```

Comment:
In the issue, there is a statement about reviewing the usage of `.toPrecision(15)` and getting rid of it if we can. What is the decision regarding it?

Reply:
please create the separate issue and consider fixing it in all the places because it had implications, here is an example https://runkit.com/1manugowda/bignumber-toprecision-issue.